packages/app-store/make/pages/setup/index.tsx

```diff
@@ -133,12 +133,23 @@ export default function MakeSetup({ inviteLink }: InferGetServerSidePropsType<ty
           </div>
         </div>
       ) : (
-        <div className="ml-5 mt-5">
-          <div>{t("install_make_app")}</div>
-          <div className="mt-3">
-            <Link href="/apps/make" passHref={true} legacyBehavior>
-              <Button>{t("go_to_app_store")}</Button>
-            </Link>
+        // Updated UI changes
+        <div className="flex h-full w-full flex-col  items-center justify-center rounded-md bg-white p-10  text-black">
+          <div className="flex w-1/3 flex-col items-center rounded-lg border border-[#E5E7EB] p-10 shadow-md">
+            <div className="flex h-16 w-16 items-center justify-center rounded-full bg-gray-100">
+              <Icon name="circle-alert" className="h-10 w-10 text-gray-800" />
+            </div>
+            <div className="mb-2 mt-4 text-2xl  text-2xl font-bold">App not installed </div>
+            <h2 className="mt-4 text-lg text-[##4B5563] ">{t("install_make_app")}</h2>
+
+            <div className="mt-5">
+              <Link href="/apps/make" passHref>
+                <Button className="border">
+                  {t("go_to_app_store")}
+                  <Icon name="arrow-up-right" />
+                </Button>
+              </Link>
+            </div>
```

Comment:
Since this is used in 4 different components, it would be better to create a single component and pass the app name to it, instead of duplicating the same thing in 4 different places

Reply:
Sure will do it

Reply:
Thanks for your work @Mihir867 , it needs some improvement. I will fix that